### PR TITLE
added sky-joker as a maintainer for zabbix modules

### PR DESCRIPTION
**Pull request by D3DeFi**
##### SUMMARY
New maintainer for Zabbix modules! yay :)

@sky-joker

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
.github/BOTMETA.yml

### CODE REVIEW

**Comment by D3DeFi** [comment]
@resmo @gundalow sorry to bother you. Can you guys merge this please?

File: .github/BOTMETA.yml [hunk 1]
```diff
@@ -260,7 +260,7 @@ files:
   $modules/monitoring/monit.py: brian-brazil
   $modules/monitoring/pagerduty.py:
     ignored: bpennypacker
-  $modules/monitoring/zabbix/: eikef D3DeFi rubentsirunyan
+  $modules/monitoring/zabbix/: eikef D3DeFi rubentsirunyan sky-joker
```
Comment (Akasurde) [inline .github/BOTMETA.yml]: @D3DeFi Thanks for the PR. Could you please create `team_zabbix` and add all names from here just like `team_ipa` or `team_vmware` ?

Comment (D3DeFi) [reply .github/BOTMETA.yml]: Thanks, changed it to team_zabbix!

**Comment by D3DeFi** [comment]
I've removed @eikef too as he has not been active with zabbix modules for a very long time now (~2years)

**Comment by Akasurde** [comment]
@D3DeFi Could you please add `cove` user in `ignored` list mentioned - https://github.com/ansible/ansible/issues/65641 ?

**Comment by D3DeFi** [comment]
> 
> 
> @D3DeFi Could you please add `cove` user in `ignored` list mentioned - #65641 ?

yup, done

**Review by Akasurde — APPROVED** [review]
LGTM

**Review by gundalow — APPROVED** [review]
@sky-joker welcome

If you've not already, please take a look at https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_checklist.html and https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_best_practices.html

**Comment by sky-joker** [comment]
Thank you all :)

**Pull request merged**